index.html

```diff
@@ -72,78 +97,48 @@ <h2 class="nomargin">Skills</h2>
 
     </section>
 
-    <div class="wr4">
+    <div class="connect-div">
 
         <h2 class="connect">Connect</h2>
 
         <a href="mailto:msb.kara.94@hmail..com">My E-Mail Addres</a>
         <a href="https://github.com/Musab-Kara">GitHub Link</a>
 
 
-        <section>
-            <h2>Leave Message</h2>
-            <form name="leave_message" id="leave_message">
-                <label for="name">Name:</label><br>
-                <input type="text" id="name" required><br>
-
-                <label for="email">E-Mail Address:</label><br>
-                <input type="email" name="email" required><br>
-
-                <label for="message">Message:</label><br>
-                <textarea name="message" required></textarea><br>
-
-                <button type="submit" id="my-btn">Submit</button>
-            </form>
-
-        </section>
-
-        <section id="message">
-            <h2>Messages</h2>
-            <ul id="messageList"></ul>
+        <section class="leaveMessage">
 
-        </section>
-        <footer>
-
-
-
-
-
-    </section>
-
-
-    <hr>
-    <h2>Connect</h2>
-
-    <a href="mailto:msb.kara.94@hmail..com">My E-Mail Addres</a>
-    <a href="https://github.com/Musab-Kara">GitHub Link</a>
-    <footer>
-
-        <section>
-            <h2>Leave a Message</h2>
-            <form name="leave_message">
-                <label for="name">Name:</label><br>
-                <input type="text" name="name" required><br>
-                <label for="email">E-Mail Address:</label><br>
-                <input type="email" name="email" required><br>
-                <label for="message">Message:</label><br>
-                <textarea name="message" required></textarea><br>
-                <button type="submit">Submit</button>
-            </form>
+            <h2>Leave Message</h2>
```

Comment:
Just a note, but I noticed that your flexbox setup on this section sometimes has weird behavior where stuff wraps in weird ways when the screen in certain sizes. It might be worthwhile fixing this with different parameters

<img width="386" alt="image" src="https://user-images.githubusercontent.com/27221245/207482369-74343e1a-4960-4277-80b3-712fd5e76dcd.png">